blockstore/src/test/java/brooklyn/location/blockstore/openstack/RackspaceVolumeManagerLiveTest.java

```diff
@@ -44,6 +44,11 @@ protected String getDefaultAvailabilityZone() {
     }
 
     @Override
+    protected char getDefaultDeviceSuffix() {
+        throw new IllegalStateException("Figure out the correct device suffix");
```

Comment:
Could add "Not implemented" so that if someone actually ends up calling it is clear that it's a problem with the code, not any of the arguments.